MetamodelUML.md

```diff
@@ -527,6 +527,7 @@ De generalisaties worden naar het volgende aspect gespecificeerd:
 
 | **Aspect**                  | **Kardinaliteit** | **Toelichting**                | **In UML 2.5**                        |    | **In EA**      | **In ...** |
 | -------------------------------- | -------- | ---------------------------------------- | ------------------------------------- | -- | -------------- | -------- |
+| **Modelelementidentificatie**                           | 1         | Identificerend metagegeven. |                                       |    |  *Tagged value* |     |
```

Comment:
Verzoek om er "identificatie" te maken. We hebben geen enkel aspect waar we de naam van het ding waar het aspect over gaat bij zetten. Waarom wijken we hier af?

Reply:
wat mij betreft ok. Wat zegt Pano? Jesse?

Reply:
@pano @Bakkej

Reply:
Ga ik doen. Het wordt 'Identificatie'

Reply:
Helemaal eens; ik had dit elders ook al opgemerkt.